src/components/search/SearchResults.tsx

```diff
@@ -100,28 +93,72 @@ const InnerSearchResultList = <T extends SearchResultsType>(
     return router.query[param]
   }
 
+  const getSearchHits = createFetchDataFn<GetSearchResults_searchQuery_hits[]>([])({
+    // Need to define this since it can't be undefined
+    chain: async ({ api }: { api: SubsocialApi }) => {
```

Comment:
Need to disable search functionality when Squid disabled

Reply:
Check config avaibleFeatures file

Reply:
Resolved in #87